### DISCUSSION TITLE
Helm charts for Openshift

### DISCUSSION THREAD

**Discussion by azeemdin** [Budibase/budibase#14882] [Ideas]
Hi,
We are exploring budibase and tried to it up on our openshift cluster. We tried to use helm charts given at following link

https://docs.budibase.com/docs/kubernetes-k8s

But these charts requires root user access specifically CouchDB and Proxy service. We managed to run the budibase with our openshift admin team but they recommended to have a version of helm charts that supports non root user. Is there any plan/suggestions to update these charts to install on openshift cluster without root access?

**Comment by AsefJafarli** [comment]
@shogunpurple is there any answer about this topic?

**Comment by bartwo** [comment]
Hi, is there any update on this? Will this be fixed for future?
